app/controllers/api/v0/reactions_controller.rb

```diff
@@ -1,25 +1,24 @@
 module Api
   module V0
-    class ReactionsController < ApplicationController
+    class ReactionsController < ApiController
       skip_before_action :verify_authenticity_token
 
       def create
-        @user = valid_user
-
-        unless @user
-          render json: { message: "invalid_user" }, status: :unprocessable_entity
+        reaction_user = load_reaction_user
+        unless reaction_user
+          render json: { error: "invalid user" }, status: :unprocessable_entity
```

Comment:
I went around the frontend code grepping with `rg reaction-creation` and double checked there are no places where we actually check the payload of the error, we just check the status code :)